src/api/wix-api-context-provider.tsx

```diff
@@ -66,32 +67,41 @@ function getWixApi(wixClient: ReturnType<typeof getWixClient>) {
                 },
                 { withBookingAllowedActions: true }
             ),
-        cancelBooking: async ({
-            _id,
-            revision,
-        }: Pick<extendedBookings.Booking, '_id' | 'revision'>) =>
-            await wixClient!.bookingsActions.cancelBooking(_id!, {
+        cancelBooking: ({ _id, revision }: Pick<extendedBookings.Booking, '_id' | 'revision'>) =>
+            wixClient!.bookingsActions.cancelBooking(_id!, {
                 revision: revision!,
             }),
-
+        getServiceAvailability: (serviceId: string) => {
+            const startDate = new Date().toISOString();
+            const endDate = new Date(
+                new Date(startDate).setMonth(new Date(startDate).getMonth() + 2)
+            ).toISOString();
```

Comment:
The `startDate` and `endDate` seem kinda arbitrary for me.

Reply:
I just put it to be the current date and 2 months forward for the sake of testing it, what defaults do you suggest? and is there a cleaner way than manipulating the `endDate` like I tried?